emails/templates/private-cloud/RequestApproval.tsx

```diff
@@ -14,7 +14,6 @@ interface EmailProp {
 
 export const RequestApprovalTemplate = ({ request }: EmailProp) => {
   if (!request) return <></>;
-
```

Comment:
to reduce the unnecessary code changes and also for readability, let's keep this new line.